app/Models/Account.php

```diff
@@ -151,6 +151,7 @@ class Account extends BaseModel
     public const FEATURE_EXPENSES = 'expenses';
     public const FEATURE_QUOTES = 'quotes';
     public const FEATURE_PURCHASE_ORDERS = 'purchase_orders';
+    public const FEATURE_PRODUCT_ALLOCATION = 'product_allocation';
```

Comment:
I do not think this is necessary. Safe to skip this.

Reply:
done